src/server/pause.js

```diff
@@ -0,0 +1,42 @@
+window.d2lTestPause = new Promise(r => window.d2lTestStart = r);
+
+const controls = `
+	<div style="padding: 1rem; background: #ddd; border-bottom: 1px solid #888; color: #222">
+		<div id="start">
+			<button id="start-button" style="margin-inline-end: 1rem;">Start</button>
+			<span>.${window.__WTR_CONFIG__.testFile.split('?')[0]}</span>
+		</div>
+		<div id="run" hidden>
+			<button id="run-button" style="margin-inline-end: 1rem;">Run</button>
+			<span id="test-name"></span>
+		</div>
+	</div>
+`;
+
+document.documentElement.insertAdjacentHTML('afterBegin', controls);
```

Comment:
When I inspect this, it's actually getting injected _above_ the `<head>` and not even inside the `<body>`. That seems... wrong.

As an aside, I'm wondering if we should make it fixed position at the bottom of the screen so that it's less likely to overlap or interfere with things under test?

Reply:
I did have it fixed to the bottom, but I realized that could actually interfere more, I think. Like if the screenshot is finding the fixture rectangle, it will always be below this, but fixed, it could overlap, right?

Reply:
Right. What just inserting it as the first child of the body work? Just don't like seeing it above the `<head>`.